core/src/main/java/org/frankframework/scheduler/job/SendMessageJob.java

```diff
@@ -68,7 +68,7 @@ public void execute() throws JobExecutionException, TimeoutException {
 		try (Message toSendMessage = getMessage() == null ? Message.nullMessage() : new Message(getMessage());
 				PipeLineSession session = new PipeLineSession()) {
 			// Set a messageId that will be forwarded by the localSender to the called adapter. Adapter and job will then share a Ladybug report.
-			session.put(PipeLineSession.CORRELATION_ID_KEY, UUIDUtil.createSimpleUUID());
+			session.put(PipeLineSession.CORRELATION_ID_KEY, MessageUtils.generateMessageId("MSG"));
```

Comment:
```suggestion
			session.put(PipeLineSession.CORRELATION_ID_KEY, MessageUtils.generateMessageId("SCHEDULED"));
```
of iets anders dat het zichtbaar is dat hij uit een scheduler komt?